sdk/cosmosdb/cosmos/test/public/functional/client.spec.ts

```diff
@@ -43,40 +43,38 @@ describe("Client Tests", function (this: Suite) {
       const client = new CosmosClient({
         endpoint: "https://faaaaaake.com",
         agent: new Agent(),
-        connectionPolicy: { enableBackgroundEndpointRefreshing: false },
```

Comment:
How is enableBackgroundEndpointRefreshing related to AAD auth refresh?

Reply:
IDK. What I see is the default connection policy changes the default behavior.....the default is true in this SDK, so it is set to false explicitly here.
https://github.com/Azure/azure-sdk-for-js/blob/7b10da87c9d476737f65ae643fcde037aa42a4a8/sdk/cosmosdb/cosmos/src/CosmosClient.ts#L74

Reply:
I see. Our tests are running fine with-out fix. Did this test start failing after fix? 
If not its unrelated lets please scope the changes to the required. Feel free to push more PR's with other changes.

Reply:
It needs to stay.  the default in Node JS only is set to true. removing can have unintended breaking.